src/gt4sd/algorithms/core.py

```diff
@@ -720,7 +912,7 @@ def get_configuration_class_with_attributes(
 
 
 class PropertyPredictor(ABC, Generic[S, U]):
-    """WIP"""
+    """TODO: Might be deprecated in future release."""
```

Comment:
what about adding a `logger.warning` message in the constructor to acknowledge the deprecation?